platform/src/test/java/org/exoplatform/platform/qa/ui/platform/ecms/EcmsManageDocumentsInSpaceTestIT.java

```diff
@@ -1,102 +1,139 @@
-    spaceManagement.acceptAInvitation(space);
-    manageLogInOut.signIn(PLFData.DATA_USER1, "gtngtn");
-    homePagePlatform.goToMySpaces();
-    ELEMENT_SPACES_LIST.find(byText(space)).click();
-    spaceManagement.goToDocumentTab();
-    $(byId("MultiUploadInputFiles")).uploadFromClasspath("eXo-Platform.png");
-    ELEMENT_LIST_DOCUMENTS_IN_SPACE.find(byText("eXo-Platform.png")).waitUntil(Condition.visible, Configuration.timeout).click();
-    documentManagement.goToPermissions();
-    $(byId(ELEMENT_CHECKBOX_MODIFY_PERMISSION.replace("{user}", space))).parent().waitWhile(Condition.not(Condition.visible),
-                                                                                            Configuration.timeout);
-    if ($(byId(ELEMENT_CHECKBOX_MODIFY_PERMISSION.replace("{user}", space))).is(Condition.checked))
-      $(byId(ELEMENT_CHECKBOX_MODIFY_PERMISSION.replace("{user}", space))).parent().click();
-    $(ELEMENT_CLOSE_BTN).click();
-    manageLogInOut.signIn(PLFData.DATA_USER2, password);
-    homePagePlatform.goToMySpaces();
-    ELEMENT_SPACES_LIST.find(byText(space)).click();
-    spaceManagement.goToDocumentTab();
-    ELEMENT_LIST_DOCUMENTS_IN_SPACE.find(byText("eXo-Platform.png"))
-                                   .waitUntil(Condition.visible, Configuration.timeout)
-                                   .contextClick();
-    ELEMENT_BUTTON_UPLOAD_NEW_VERSION.shouldNotBe(Condition.visible);
-    ELEMENT_BUTTON_CREATE_NEW_VERSION.shouldNotBe(Condition.visible);
-    ELEMENT_LIST_DOCUMENTS_IN_SPACE.find(byText("eXo-Platform.png")).waitUntil(Condition.visible, Configuration.timeout).click();
-    ELEMENT_BUTTON_UPLOAD_NEW_VERSION.shouldNotBe(Condition.visible);
-    ELEMENT_BUTTON_CREATE_NEW_VERSION.shouldNotBe(Condition.visible);
-    manageLogInOut.signIn(PLFData.DATA_USER1, "gtngtn");
-    homePagePlatform.goToMySpaces();
-    spaceManagement.deleteSpace(space, false);
-
-  }
+    SpaceManagement spaceManagement;
+
+    SpaceHomePage spaceHomePage;
+
+    NavigationToolbar navigationToolbar;
+
+    SpaceSettingManagement spaceSettingManagement;
+
+    SiteExplorerHome siteExplorerHome;
+
+    CreateNewDocument createNewDocument;
+
+    ManageLogInOut manageLogInOut;
+
+    HomePagePlatform homePagePlatform;
+
+    DocumentManagement documentManagement;
+
+    @BeforeEach
+    public void setupBeforeMethod() {
+        info("Start setUpBeforeMethod");
+        manageLogInOut = new ManageLogInOut(this);
+        spaceManagement = new SpaceManagement(this);
+        spaceHomePage = new SpaceHomePage(this);
+        siteExplorerHome = new SiteExplorerHome(this);
+        createNewDocument = new CreateNewDocument(this);
+        navigationToolbar = new NavigationToolbar(this);
+        manageLogInOut.signInCas("john", "gtngtn");
+        spaceSettingManagement = new SpaceSettingManagement(this);
+        homePagePlatform = new HomePagePlatform(this);
+        documentManagement = new DocumentManagement(this);
```

Comment:
Indentation is not good (it was ok before the changes).

Reply:
That does not fix the indentation issue here.

Reply:
The line exists since Ilyes has created the class
![Screenshot from 2019-06-26 16-56-52](https://user-images.githubusercontent.com/50202793/60190927-bfcabe00-9833-11e9-91f6-9848ada2e695.png)

Reply:
Yes but changed the indentation, it is now 4 spaces instead of 2 spaces.

Reply:
Fixed.